TROUBLESHOOTING.md

```diff
@@ -104,22 +107,23 @@ To fix this, you may either:
 **Solution**
 
 Your base64'ed PAT token has a new line at the end, it needs to be created without a `\n` added, either:
-* `echo -n $TOKEN | base64`
-* Create the secret as described in the docs using the shell and documented flags
+
+- `echo -n $TOKEN | base64`
+- Create the secret as described in the docs using the shell and documented flags
```

Comment:
I'm fine with the newline but I don't see the value of switching to `*` from `-`